src/Tribe/JSON_LD/Order.php

```diff
@@ -101,10 +101,17 @@ public function add_ticket_data( $data, $args, $post ) {
 	 * @return object
 	 */
 	public function get_offer( $ticket, $event ) {
+		$price = $ticket->price;
+
+		// JSON-LD can't have free as a price
+		if ( strpos( strtolower( trim( $price ) ), 'free' ) !== false ) {
```

Comment:
In the-events-calendar, the `Free` value is set via `__( 'Free', 'the-events-calendar' )`. Should this string be translated as well?